cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
ten plik powinien być pusty?

Reply:
To jest ten, który generuje się automatycznie po pierwszym odpaleniu testów. Można tu wrzucić konfigurację. Nie bardzo wiedziałem co tu na chwilę obecną ustawiać, więc pozostał pusty